### PR TITLE
Update blog index and blog post

### PR DESCRIPTION
**Pull request by julianbenegas**

### CODE REVIEW

**Review by adamsoffer — CHANGES_REQUESTED** [review]
@julianbenegas looks amazing 😍. Can we just change the heading to `Blog` and subheading to `Welcome to the Livepeer.com blog.`?

And I think splicing the most recent post to use as the featured post like you have is fine for now 👍

File: packages/www/pages/blog/[slug].tsx [hunk 1]
```diff
@@ -59,109 +65,116 @@ const Post = ({
...
-                  day: "numeric"
-                })}
-              </Box>
-              <Box sx={{ mx: 3, width: "1px", height: 16, bg: "grey" }} />
-              <Box>{category.title}</Box>
-            </Flex>
-            <h1 sx={{ fontSize: [32, 32, 48], my: 3 }}>{title}</h1>
-            <Flex sx={{ alignItems: "center" }}>
-              <Flex sx={{ alignItems: "center" }}>
+            {category.title}
```
Comment (adamsoffer) [inline packages/www/pages/blog/[slug].tsx]: @julianbenegas can we link this to the category page?

Comment (julianbenegas) [reply packages/www/pages/blog/[slug].tsx]: Sure thing!

**Review by adamsoffer — APPROVED** [review]
LGTM! 🙏 🤘

**Pull request merged**